src/nvim/move.c

```diff
@@ -134,7 +134,11 @@ void update_topline(void)
 
   // If there is no valid screen and when the window height is zero just use
   // the cursor line.
-  if (!screen_valid(true) || curwin->w_height == 0) {
+  if (!screen_valid(true))
+    return;
```

Comment:
what for this extra return?

Reply:
I think this slipped in during rebase :sweat_smile: .